support-frontend/assets/pages/contributions-landing/components/ContributionAmount.tsx

```diff
@@ -187,12 +191,23 @@ function ContributionAmount(props: PropTypes) {
 								props.contributionType,
 							)
 						}
-						onBlur={() =>
-							!!otherAmount &&
-							trackComponentClick(
-								`npf-contribution-amount-toggle-${props.countryGroupId}-${props.contributionType}-${otherAmount}`,
-							)
-						}
+						onBlur={() => {
+							if (otherAmount) {
+								trackComponentClick(
+									`npf-contribution-amount-toggle-${props.countryGroupId}-${props.contributionType}-${otherAmount}`,
+								);
+
+								const otherAmountNumber = parseInt(otherAmount, 10);
+
+								if (typeof otherAmountNumber === 'number') {
```

Comment:
Is the idea here to stop `NaN`s being sent? Unfortunately `typeof NaN` is `'number'`

Reply:
Indeed it is, good spot, I'll use `isNaN()` to catch `NaN` instead!